examples/nodejs/cache/doc-example-files/cheat-sheet-main.ts

```diff
@@ -1,10 +1,9 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
-import {CacheClient, Configurations, CredentialProvider} from '@gomomento/sdk';
+import {CacheClient, Configurations} from '@gomomento/sdk';
 
 async function main() {
   const cacheClient = await CacheClient.create({
     configuration: Configurations.Laptop.v1(),
-    credentialProvider: CredentialProvider.fromEnvironmentVariable('MOMENTO_API_KEY'),
```

Comment:
`configuration` is actually optional now too; I just never went and updated the docs. (so thank you for being diligent about it!)

I think we should leave plenty of examples of how to explicitly set these so that people will know, but in the examples where we omit them, we should omit both. I think this file is a good place to omit both, and whatever shows up in the main README is another good spot.  Maybe also in the most basic example file for each product. But probably the rest of the places (e.g. the dictionary example) I'd leave the explicit versions in.

You can use your judgment on which files to update vs leave explicit, but let's go ahead and remove the explicit `configuration` option from any of the ones where you remove `credentialProvider`.